config/src/test/java/net/consensys/cava/config/TomlBackedConfigurationTest.java

```diff
@@ -439,8 +439,11 @@ void writeConfigurationToToml() throws Exception {
       + "\"\" = \"emptykey\"\n"
       + "\n"
       + "[deepmap.deeper]\n"
-      + "farewell = \"goodbye\"\n";
+      + "farewell = \"goodbye\"\n").replace("\n", System.lineSeparator());
     // @formatter:on
+
+    System.out.println(expected.replace("\n", "\\n").replace("\r", "\\r"));
```

Comment:
Should have been deleted?

Reply:
🤦‍♂️ sorry I'm tired. I'll give it the once over.